doc/user/content/integrations/aws-msk.md

```diff
@@ -0,0 +1,148 @@
+
+The process to connect Materialize to Amazon MSK consists of the following steps:
+1. #### Create an Amazon MSK cluster
+    If you already have an Amazon MSk cluster set up, then you can skip this step.
+
+    a. Sign in to the AWS Management console and open the [Amazon MSK console](https://us-east-1.console.aws.amazon.com/msk/)
+
+    b. Choose **Create cluster**
+
+    c. Enter a cluster name, and leave all other settings unchanged
+
+    d. From the table under **All cluster settings**, copy the values of the following settings and save them because you need them later in this tutorial: **VPC**, **Subnets**, **Security groups associated with VPC**
+
+    e. Choose **Create cluster**
+
+    **Note:** This creation can take about 15 minutes.
+
+2. #### Make the cluster public and enable SASL
+    ##### Turn on SASL
+    a. Navigate to the [Amazon MSK console](https://us-east-1.console.aws.amazon.com/msk/)
+
+    b. Choose the MSK cluster you just created in Step 1
+
+    c. Click on the **Properties** tab
+
+    d. In the **Security settings** section, choose **Edit**
+
+    e. Check the checkbox next to **SASL/SCRAM authentication**
+
+    f. Click **Save changes**
+
+    You can find more details about updating an MSK cluster's security configurations [here](https://docs.aws.amazon.com/msk/latest/developerguide/msk-update-security.html).
+
+    ##### Create a symmetric key
+    a. Now go to the [Amazon KMS (Key Management Service) console](https://us-east-1.console.aws.amazon.com/kms)
+
+    b. Click **Create Key**
+
+    c. Choose **Symmetric** and click **Next**
+
+    d. Give the key and **Alias** and click **Next**
+
+    e. Under Administrative permissions, check the checkbox next to the **AWSServiceRoleForKafka** and click **Next**
+
+    f. Under Key usage permissions, again check the checkbox next to the **AWSServiceRoleForKafka** and click **Next**
+
+    g. Click on **Create secret**
+
+    h. Review the details and click **Finish**
+
+    You can find more details about creating a symmetric cmk [here](https://docs.aws.amazon.com/kms/latest/developerguide/create-keys.html#create-symmetric-cmk).
+
+    ##### Store a new Secret
+    a. Go to the [AWS Secrets Manager console](https://us-east-1.console.aws.amazon.com/secretsmanager/)
+
+    b. Click **Store a new secret**
+
+    c. Choose **Other type of secret** (e.g. API key) for the secret type
+
+    d. Under **Key/value pairs** click on **Plaintext**
+
+    e. Paste the following in the space below it and replace `<your-username>` and `<your-password>` with the username and password you want to set for the cluster
+      ```
+        {
+          "username": "<your-username>",
+          "password": "<your-password>"
+        }
+      ```
+
+    f. On the next page, give a **Secret name** that starts with **AmazonMSK_**
+
+    g. Go forward to the next steps and finish creating the secret. Once created, record the ARN (Amazon Resource Name) value for your secret
```

Comment:
Do users need to specify their new "Encryption Key" below the Plaintext field?
![image](https://user-images.githubusercontent.com/11527560/163880750-f776dd10-2188-4df4-9165-afc25393cf64.png)